src/hooks/usePropsGetters/useNextQuestionButtonProps.ts

```diff
@@ -24,10 +27,22 @@ export default function useNextQuestionButtonProps(
       'aria-describedby': buttonDisabled ? 'next-button-help' : '',
       type: 'button',
       onClick: () => {
+        if (currentQuestion?.type === 'open' && onOpenQuestionAnswered && currentQuestionId) {
+          const value = quizLocalState.answerInputs[currentQuestionId]?.value;
+
+          if (value && typeof value === 'string') {
+            onOpenQuestionAnswered(value, currentQuestion);
```

Comment:
@Gull-Dev I believe this is already supported on `onQuizNextQuestion` callback inside `callbacks` passed to `IQuizProps`

And the callback takes these two parameters same as your example `currentQuestion, answer: currentAnswerInput`

check the call of `onQuizNextQuestion` inside `useQuizNextClick` hook